packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md

```diff
@@ -1,27 +1,30 @@
 
 ### Use SAML toolkits
 
-The following open source and paid tool kits can help you implement SAML 2.0 for your existing app. These different language kits help you create the WebSSO Profile for Service Providers:
+To quickly build your SSO integration, use an open source or paid tool kit to implement SAML 2.0. Use these different language kits to create the Service Provider WebSSO profile:
 
-* .NET Framework 4.5 or later: [Sustainsys.Saml2](https://github.com/Sustainsys/Saml2) (formerly Kentor Authentication Services)
-* .NET Framework 4.0 or earlier: [ComponentSpace SAML 2.0 for ASP.NET and ASP.NET Core](https://www.componentspace.com/) (paid software)
-* Java: [OpenSAML](https://wiki.shibboleth.net/confluence/display/OS30/Home), which is part of the [Shibboleth Development Project](https://www.shibboleth.net/)
+* .NET framework 4.5 or later: [Sustainsys.Saml2](https://github.com/Sustainsys/Saml2) (Kentor authentication services)
+* .NET framework 4.0 or earlier: [ComponentSpace SAML 2.0 for ASP.NET and ASP.NET Core](https://www.componentspace.com/) (paid software)
+* Java: [OpenSAML](https://wiki.shibboleth.net/confluence/display/OS30/Home) (part of the [Shibboleth Development Project](https://www.shibboleth.net/))
 * Java: [Spring Security SAML](/code/java/spring_security_saml)
 * Python: [PySAML2](/code/python/pysaml2)
 * Ruby: [Ruby-SAML](https://github.com/onelogin/ruby-saml)
 
->**Note:** Okta doesn't own or maintain these toolkits, however, some documentation is provided to help you use them with Okta.
+>**Note:** Okta doesn't own or maintain these toolkits.
 
 ### Gather SAML attributes
 
-You need to obtain SAML integration fields before you create an app integration instance in Okta.
+Obtain SAML integration attributes before you create an app integration instance in Okta.
 
 In a SAML integration, Okta is the Identity Provider (IdP), and your app is the Service Provider (SP).
 
-1. Determine the default Assertion Consumer Service (ACS) URL for your integration. This is often referred to as the SP sign-in URL. This is the endpoint on your application where the SAML responses are posted.
-1. Find your Audience URI. This is sometimes referred to as the SP entity ID or the entity ID of your app.
-1. (Optional) Set up a Default Relay State page, where users land after they successfully sign in to the SP using SAML. This must be a valid URL.
-1. Gather any required SAML attributes. You can choose to share Okta user profile values as SAML attributes with your app.
+1. Determine the default assertion consumer service (ACS) URL for your integration. This is often referred to as the SP sign-in URL. This is the endpoint on your app where the SAML responses are posted.
+1. Find your audience URI. This is sometimes referred to as the SP entity ID or the entity ID of your app.
+1. Set up a default relay state page, where users land after they successfully sign in to the SP using SAML. This must be a valid URL. (Optional)
```

Comment:
```suggestion
1. Optional. Set up a default relay state page, where users land after they successfully sign in to the SP using SAML. This must be a valid URL. 
```

Reply:
On the prod side, we recommend putting `Optional.` at the beginning of the step, no parentheses. Not sure if the dev docs recommendation is different?

Reply:
Acrolinx doesn't like it if the list isn't consistent. For this numbered list, Acrolinx is looking for an action verb at the beginning of every item. I can try to use `Optional.` and check the score again. I feel pretty bad that we have to lowercased ".NET framework" for the score to get over 80 :-(

Reply:
Just checked. I'm under 80 if I use `Optional.` at the beginning. Here's the Acrolinx message:
![image](https://github.com/user-attachments/assets/1aaa25c1-37e9-44b5-853f-f46c6eab41f5)